src/core/mavsdk_impl.cpp

```diff
@@ -305,7 +325,7 @@ uint8_t MavsdkImpl::get_own_component_id() const
 
         case Mavsdk::Configuration::CompanionComputer:
             // It's at least a possibility that we are bridging MAVLink traffic.
-            return MAV_COMP_ID_UDP_BRIDGE;
+            return 99;
```

Comment:
Server doesn't work because of this:

void SystemImpl::process_heartbeat(const mavlink_message_t& message)
{
    // FIXME: for now we ignore heartbeats from UDP_BRIDGE because that's just
    // confusing since it doesn't mean a vehicle is connected.
    if (message.compid == MAV_COMP_ID_UDP_BRIDGE) {
        return;
    }
...

I think we should be able to set component ID when we create mavlink FTP server. This way it would be possible to have more then one server on the same companion computer.

Reply:
We should actually be able to remove this hack in `process_heartbeat` because we now use `has_autopilot()` to check if something is connected.

Reply:
I'm removing it here: https://github.com/mavlink/MAVSDK/pull/828

Reply:
Now it works!